test_runner/fixtures/endpoint/http.py

```diff
@@ -21,3 +21,8 @@ def database_schema(self, database: str):
         res = self.get(f"http://localhost:{self.port}/database_schema?database={database}")
         res.raise_for_status()
         return res.text
+
+    def extensions(self):
```

Comment:
```suggestion
    def installed_extensions(self):
```
For the sake of consistency